README.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is this repository? <a name="what_is_it"></a>
 This repository contains tools, templates, and information for assembling, debugging, testing, 
-and running your custom inference models and custom tasks with DataRobot.
```

Comment:
We might need for content for notebook environments here

Reply:
Left a comment to add a sentence about notebook environments to this section. I can open up a separate PR later to update the README to have a Notebooks environment reference section that links to the Notebook documentation (will have to do this after the public preview docs are published)

Reply:
Sounds good.